diffyscan/diffyscan.py

```diff
@@ -105,15 +105,15 @@ def run_source_diff(
 ):
     logger.divider()
     logger.okay("Contract", contract_address_from_config)
-    logger.okay("Blockchain explorer Hostname", config["explorer_hostname"])
+    logger.okay("Blockchain explorer Hostname", get_explorer_hostname(config))
     logger.okay("Repo", config["github_repo"]["url"])
     logger.okay("Repo commit", config["github_repo"]["commit"])
     logger.okay("Repo relative root", config["github_repo"]["relative_root"])
 
     logger.divider()
 
     logger.info(
-        f"Fetching source code from blockchain explorer {config['explorer_hostname']} ..."
+        f"Fetching source code from blockchain explorer {get_explorer_hostname(config)} ..."
```

Comment:
Do we need this log at all? We have already logged Blockchain explorer above in the code.